tests/cases/fourslash/server/importCompletions_importsMap3.ts

```diff
@@ -0,0 +1,36 @@
+/// <reference path="../fourslash.ts"/>
+
+// @Filename: /tsconfig.json
+//// {
+////   "compilerOptions": {
+////     "module": "nodenext",
+////     "rootDir": "src",
+////     "outDir": "dist"
+////   }
+//// }
+
+// @Filename: /package.json
+//// {
+////   "type": "module",
+////   "imports": {
+////     "#internal/": "./dist/internal/"
+////   }
+//// }
+
+// @Filename: /src/internal/foo.ts
+//// export function something(name: string) {}
+
+// @Filename: /src/a.ts
+//// import {} from "/*1*/";
+//// import {} from "#internal//*2*/";
+
+verify.completions({
+    marker: ["1"],
+    exact: ["#internal"],
+    isNewIdentifierLocation: true,
+});
+verify.completions({
+    marker: ["2"],
+    exact: ["foo.js"],
```

Comment:
This currently fails. I took this test case from https://github.com/microsoft/TypeScript/pull/55015/files#diff-c8e8521e91d6946ce7869359a143f8cf7b65ee15703a4e04c7d73dcbaa5682a4 but I think that perhaps it should just be removed/tweaked in both places. A trailing slash is already not supported by node@>=17 and it was deprecated before that. There is little value in supporting this - in my opinion, it would be better to discourage this completely.

In addition to that, I noticed that the previous segment gets suggested in a situation like this today:
```json
{
    "name": "pkg_test",
    "exports": {
        "./test/": "./"
    }
}
```

```ts
import { bazinga } from 'pkg_test/test//*1*/'
```

At marker `1` I get `test` again, it's imho a clear bug and I can fix it here - after we agree on not supporting this old deprecated pattern at all.